test-case/check-pause-release-suspend-resume.sh

```diff
@@ -22,6 +22,30 @@
 ## Expect result:
 ##    no errors occur
 ##
+##
+## To run test manually:
+## Preconditions
+##     1. Device has ability to fully suspend.
+##         - BYTs cannot enter necessary suspend state.
+## Test Description
+##     Check for errors during test cycle of:
+##         playback -> pause -> suspend -> resume -> release cycles
```

Comment:
here, can we change `playback` to `playback/capture`?

Reply:
will do.